legend-sdlc-generation-service/src/main/resources/generation/service/ServiceExecutionClassGenerator.ftl

```diff
@@ -1,18 +1,17 @@
-<#--
-Copyright 2022 Goldman Sachs
+// Copyright 2022 Goldman Sachs
```

Comment:
I have two questions on this, and I genuinely don't know the answer to either one.

First, do we need to keep the previous license/copyright statement? Or does the license/copyright that's part of the template constitute a license/copyright on the template itself?

Second, is 2022 the right year for the copyright? Presumably, people will be generating code from this template in 2023 and later. Should the year for generated copyrights be the year in which the code is generated (meaning the year needs to be templatized)? Or should be the year be the same as the year of the copyright of the template itself (meaning 2022 is correct)?